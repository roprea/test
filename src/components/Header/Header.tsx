import { memo } from 'react';
import type { HTMLAttributes } from 'react';

function Header(props: Props) {
  return <header {...props}>App Header</header>;
}

type Props = HTMLAttributes<HTMLElement>;

export default memo(Header);
