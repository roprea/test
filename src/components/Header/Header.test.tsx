import { basicTestRunner } from 'test-utils/basic-test-runner';
import { BrowserRouter } from 'react-router-dom';
import Header from './Header';

describe('Header', () => {
  basicTestRunner(<Header />, () => ({}), { wrapper: BrowserRouter });
});
