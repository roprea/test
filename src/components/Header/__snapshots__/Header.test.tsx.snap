// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Basic component behaviour should match snapshot 1`] = `
<div>
  <header>
    App Header
  </header>
</div>
`;
